src/app/ReadPrepareParams.h

```diff
@@ -41,17 +42,19 @@ typedef void (*OnResubscribePolicyCB)(uint32_t aNumCumulativeRetries, uint32_t &
 struct ReadPrepareParams
 {
     SessionHolder mSessionHolder;
-    EventPathParams * mpEventPathParamsList         = nullptr;
-    size_t mEventPathParamsListSize                 = 0;
-    AttributePathParams * mpAttributePathParamsList = nullptr;
-    size_t mAttributePathParamsListSize             = 0;
-    EventNumber mEventNumber                        = 0;
-    System::Clock::Timeout mTimeout                 = kImMessageTimeout;
-    uint16_t mMinIntervalFloorSeconds               = 0;
-    uint16_t mMaxIntervalCeilingSeconds             = 0;
-    bool mKeepSubscriptions                         = true;
-    bool mIsFabricFiltered                          = false;
-    OnResubscribePolicyCB mResubscribePolicy        = nullptr;
+    EventPathParams * mpEventPathParamsList                 = nullptr;
+    size_t mEventPathParamsListSize                         = 0;
+    AttributePathParams * mpAttributePathParamsList         = nullptr;
+    size_t mAttributePathParamsListSize                     = 0;
+    DataVersionFilterParams * mpDataVersionFilterParamsList = nullptr;
+    size_t mDataVersionFilterParamsListSize                 = 0;
```

Comment:
Again, this should be a Span.

Reply:
https://github.com/project-chip/connectedhomeip/issues/14915